tern/report/errors.py

```diff
@@ -52,6 +52,10 @@
     '''{default_shell}\n'''
 unknown_content = '''Unknown content included in layer {files}. Please ''' \
     '''analyze these files separately\n'''
+invalid_raw_image = '''Invalid raw image provided: '{image}' - Check ''' \
+    '''that path to raw image is correct and in tar archive format.'''
+incorrect_raw_option = '''Raw image detected. Expected docker image ''' \
+    '''with image:tag format.'''
```

Comment:
The API will actually take an image string of the form `image@<digest-type>:<digest>` so maybe make this more generic?

Reply:
@nishakm How does "Expected docker image but detected file in tar archive format." sound?